### PR TITLE
Docker Release for Avail Light

### PR DESCRIPTION
**Pull request by obrienalaribe**
- include dockerfile.release 
- update CI to push images to dockerhub

### CODE REVIEW

**Review by sh3ll3x3c — APPROVED** [review]
lgtm

File: Dockerfile.release [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+RUN cp /app/target/$BUILD_PROFILE/avail-light /app/avail-light
+
+FROM rust:1-slim-buster as run
+WORKDIR /app
+RUN apt-get update && apt-get install -y ca-certificates
+
+COPY --from=builder /app/avail-light /usr/local/bin
+
+EXPOSE 7000 37000
+CMD ["/usr/local/bin/avail-light", "-n", "goldberg"]
```
Comment (moraesjeremias) [inline Dockerfile.release:23]: Why not entrypoint?

**Review by moraesjeremias — COMMENTED** [review]
LGTM, left a small comment.

File: Dockerfile.release (hunk 1 above)
Comment (obrienalaribe) [reply Dockerfile.release:23]: Good call, can switch it to entrypoint

Comment (moraesjeremias) [reply Dockerfile.release:23]: I'll revisit this later.

**Pull request merged**